docs/sources/datasources/loki/configure-loki-data-source.md

```diff
@@ -114,9 +114,13 @@ Each derived field consists of the following:
 
 - **Type** - Defines the type of the derived field. It can be either:
 
-  - **Regex**: A regular expression to parse a part of the log message and capture it as the value of the new field. Can contain only one capture group.
+{{% admonition type="note" %}}
```

Comment:
is it worth using the type of label "caution" which renders in yellow?

Reply:
good call, let's do that! thank you!